webapp/components/_new/features/Admin/Badges/BadgesSection.vue

```diff
@@ -8,7 +8,7 @@
         @click="toggleBadge(badge)"
         :class="{ badge, inactive: !badge.isActive }"
       >
-        <img :src="badge.icon | proxyApiUrl" :alt="badge.description" />
+        <img :src="'/api' + badge.icon" :alt="badge.description" />
```

Comment:
I prefer a filter here. This way we could also omit the leading slash on the `icon` attribute. (See File above in review)

Why I prefer a filter? If we change the endpoint of the api e.g. from `domain/api` to `api.domain` we have a trillion occurrences, we have to update.

rather do:
```suggestion
        <img :src="badge.icon | apiUrl " :alt="badge.description" />
```

This way you can also enforce the `/` if not present. Preferable with https://developer.mozilla.org/en-US/docs/Web/API/URL/URL .

Reply:
Could also be `backenUrl` as filter of `fromBackend` - whatever suits best

Reply:
Filters have been removed in Vue V3: https://v3-migration.vuejs.org/breaking-changes/filters